chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
Also, in a dynamic language like JS, I think having variable declarations in general makes no sense. E.g. Ruby and Python do not have variable declarations and everything is function scope, which mostly makes sense. Then you can reuse the same variable somewhere else, without worrying about declarations. I don't get it why they didn't use the chance with ES6 to remove declarators completely and instead added 2 more, which make their usage even more ambiguous. I think just sticking to `var` everywhere should be preferable and cause least headaches (however, double declarations cause problems in strict mode, so using `let` which has the hard enforcement also makes sense).